lib/components/src/form/button.js

```diff
@@ -0,0 +1,12 @@
+import glamorous from 'glamorous';
+
+export default glamorous
+  .button({
+    border: 'solid 1px rgba(0, 0, 0, 0.2)',
+    background: 'rgba(255, 255, 255, 0.5)',
+    padding: '5px 10px',
+    borderRadius: '4px',
+    color: 'rgba(0, 0, 0, 0.5)',
+    outline: 'none',
```

Comment:
Let's either not remove it in a generic button, or provide something instead, using `':focus`: {...styles}`

Reply:
We could add some `:hover` and `:active` states as well